<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>develop</class>
 <widget class="QDialog" name="develop">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>708</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>-10</y>
     <width>131</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正像素24</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>进化树</string>
   </property>
  </widget>
  <widget class="QPushButton" name="littleCat">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>140</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正像素24</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>可爱的小猫团</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>51</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/develop);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="smallCat">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>200</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正像素24</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>淘气的小猫咪</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dilu">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>270</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正像素24</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>伪迪路兽</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fireCat">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>340</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正像素24</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>暴脾气的小火喵</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ghostCat">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>340</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正像素24</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>游魂野喵</string>
   </property>
  </widget>
  <widget class="QPushButton" name="iceCat">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>340</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正像素24</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>很高冷的小冰喵</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正像素24</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>蛋</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>70</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正像素24</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>开天辟地的黑心皮蛋</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正像素24</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>幼年期（Lv.5）</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正像素24</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>成长期(Lv.10)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正像素24</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>成熟期(Lv.15)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正像素24</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>完全体</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正像素24</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>究极体</string>
   </property>
  </widget>
  <widget class="QPushButton" name="redCoat">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>410</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正像素24</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>红衣</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ha">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>410</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正像素24</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>蛤莱士</string>
   </property>
  </widget>
  <widget class="QPushButton" name="old">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>410</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正像素24</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>长者</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正像素24</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>(Lv.20)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正像素24</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>(Lv.30)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>170</y>
     <width>5</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(116, 116, 116);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>230</y>
     <width>5</width>
     <height>42</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(116, 116, 116);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>300</y>
     <width>5</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(116, 116, 116);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>370</y>
     <width>5</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(116, 116, 116);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>370</y>
     <width>5</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(116, 116, 116);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>370</y>
     <width>5</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(116, 116, 116);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>350</y>
     <width>51</width>
     <height>5</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(116, 116, 116);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>350</y>
     <width>60</width>
     <height>5</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(116, 116, 116);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>110</y>
     <width>5</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(116, 116, 116);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="tips">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>470</y>
     <width>471</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正像素24</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>要完成此阶段进化 请使用钢琴弹奏下面旋律：</string>
   </property>
  </widget>
  <widget class="QLabel" name="songNote">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>510</y>
     <width>411</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/piano/mission);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="tips_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>560</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正像素24</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>(可以猜猜这段旋律是什么 不过没有奖励)</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
